pydoctor/epydoc/markup/epytext.py

```diff
@@ -131,11 +131,18 @@
 #   4. helpers
 #   5. testing
 
-from typing import Any, List, Optional, Sequence, Union, cast, overload
+from typing import Any, Iterable, List, Optional, Sequence, Union, cast, overload
 import re
 
-from twisted.web.template import CharRef, Tag, tags
-from pydoctor.epydoc.doctest import colorize_doctest
+from docutils import utils
+from docutils.nodes import (Node, Text, document, inline, paragraph, literal, reference, 
```

Comment:
We should probably use `from docutils import nodes` instead, this would avoid a lot of risk to create bugs related to variable names overrides.

Reply:
Done